src/renderers/dom/ReactDOM.js

```diff
@@ -35,6 +37,8 @@ var React = {
   render: render,
   unmountComponentAtNode: ReactMount.unmountComponentAtNode,
   version: ReactVersion,
+  addDevtool: ReactDOMDevtools.addDevtool,
+  removeDevtool: ReactDOMDevtools.removeDevtool,
```

Comment:
Just removed them, as per conversation, we can figure out how to expose them later.